src/index.js

```diff
@@ -1,7 +1,168 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import Map from './map.js'
+import leafletPip from  '@mapbox/leaflet-pip'
+import L from 'leaflet'
+import borderData from './border.js'
+
+const wholeThang = {
+  display: "flex",
+
+}
+const img = {
+  height : "500px"
+}
+
+
+class App extends React.Component {
+  state = { 
+    markerPosition: { lat: 44.5588,lng: -72.5778 },
+    mapCenter: {lat: 44.5588, lng: -72.5778},
+    town: {}  
+  };
+  
+  startGame = () => {
+    document.getElementById("start-button").disabled = true
+    document.getElementById("guess-button").disabled = false
+    document.getElementById("quit-button").disabled = false
+    let randoLat = this.getRandoLat()
+    let randoLng = this.getRandoLng()
+    let layerArray = leafletPip.pointInLayer ([randoLng, randoLat],L.geoJSON(borderData))
+    while (layerArray.length === 0){
+      randoLat = this.getRandoLat()
+      randoLng = this.getRandoLng()
+      layerArray = leafletPip.pointInLayer ([randoLng, randoLat],L.geoJSON(borderData))
+    }
+    this.setState({
+      mapCenter:{
+        lat: randoLat,
+        lng:randoLng
+      },
+      markerPosition:{
+        lat: randoLat,
+        lng:randoLng
+    }
+  })
+  }
+  componentDidMount = () => {
+    document.getElementById("start-button").disabled = false
+    document.getElementById("guess-button").disabled = true
+    document.getElementById("quit-button").disabled = true
+  }
+  getRandoLat = () => {
+    let lat = Math.random() * (45.005419 - 42.730315) + 42.730315;
+    return lat;
+  }
+ 
+  getRandoLng = () => {
+    let lng = (Math.random() * (71.510225 - 73.35218) + 73.35218) * -1;
+    return lng;
+  }
+
+  showLocation = () => {
+    document.getElementById('latLng').textContent = `Lat: ${this.state.markerPosition.lat} Lng: ${this.state.markerPosition.lng}`
+    document.getElementById('start-button').disabled = false
+    this.lookupLocation()
+  }
+
+  lookupLocation = () => {
+    fetch(`https://nominatim.openstreetmap.org/reverse/?lat=${this.state.markerPosition.lat}&lon=${this.state.markerPosition.lng}&format=json`)
+    .then(response => {
+      return response.json()
+    }) .then(obj => {
+      console.log(obj)
+      document.getElementById('town').textContent = `Town: ${obj.address.village||obj.address.city||obj.address.hamlet}`
+      document.getElementById('county').textContent = `County: ${obj.address.county}`
+    })
+
+  }
+  
+  
+  moveNorth = () => {
```

Comment:
This is a good first pass on the move buttons. in a bit (once you've finished the rest of the backlog) you might want to go back and refactor these down to a single function since the only difference between them is the direction (lat or long) and the offset (+ or -)